assets/js/modules/analytics-4/components/dashboard/DashboardOverallPageMetricsWidgetGA4.stories.js

```diff
@@ -199,7 +199,6 @@ ZeroData.args = {
 		}
 	},
 };
-ZeroData.scenario = {};
```

Comment:
I removed some scenarios here which were likely not added intentionally and don't really add much value. These were some of the most unstable likely because of text resizing we do in smaller viewports. Even still, we do cover it but we don't need scenarios for every variant.